### PR TITLE
Fix automated docs [wip]

### PR DESCRIPTION
**Pull request by joshua-mir**
autogenerated config docs included

### CODE REVIEW

File: Configuring-Parity-Ethereum.md [hunk 1]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
 
 ## Default config.toml
 
-The following is a representation of a configuration file with all default values.
+The following is a representation of a configuration file with all default values (*note: the `[stratum]` section is not present by default, and including it in your config currently enables stratum*).
 
-```toml
-[parity]
-mode = "last"
+```toml[parity]
```
Comment (5chdn) [inline Configuring-Parity-Ethereum.md]: missing line break

File: Configuring-Parity-Ethereum.md [hunk 2]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
+mode = last
 mode_timeout = 300
 mode_alarm = 3600
-auto_update = "critical"
-release_track = "current"
-public_node = false
+auto_update = critical
+auto_update_delay = 100
+auto_update_check_frequency = 20
+release_track = current
```
Comment (5chdn) [inline Configuring-Parity-Ethereum.md]: quotes missing?

File: Configuring-Parity-Ethereum.md [hunk 3]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
+`false`
+
+## mode
+Set the operating mode. MODE can be one of: last - Uses the last-used mode, active if none; active - Parity continuously syncs the chain; passive - Parity syncs initially, then sleeps and wakes regularly to resync; dark - Parity syncs only when the JSON-RPC is active; offline - Parity doesn't sync.
+#### Command line option 
+ `--mode`
+#### Config file option 
+```toml
+[parity]
+mode = last
```
Comment (5chdn) [inline Configuring-Parity-Ethereum.md]: quotes?

File: Configuring-Parity-Ethereum.md [hunk 4]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
+## mode-timeout
+Specify the number of seconds before inactivity timeout occurs when mode is dark or passive
+#### Command line option 
+ `--mode-timeout`
+#### Config file option 
+```toml
+[parity]
+mode_timeout = 300
+```
+#### Default Value
```
Comment (5chdn) [inline Configuring-Parity-Ethereum.md]: that's a lot of headlines. maybe one headline per command is sufficient?

File: Configuring-Parity-Ethereum.md [hunk 5]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
+Does nothing; checking if clock is sync with NTP servers is now done on the UI.
+#### Command line option 
+ `--ntp-servers`
+#### Default Value 
+`null`
+
+# misc
+
+## unsafe-expose
+All servers will listen on external interfaces and will be remotely accessible. It's equivalent with setting the following: --[ws,jsonrpc,ui,ipfs-api,secretstore,stratum,dapps,secretstore-http]-interface=all --*-hosts=all    This option is UNSAFE and should be used with great care!
```
Comment (5chdn) [inline Configuring-Parity-Ethereum.md]: this somehow breaks markdown formatting

File: Configuring-Parity-Ethereum.md [hunk 6]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
-- `dev`: uses [dev chain specifications](Private-development-chain) with [Instant-seal](Pluggable-Consensus#instant-seal) consensus engine. The gas price is set to 0.
-- `dev-insecure`: uses the same configuration as `dev`, plus sets the flag `no_consensus`, allows all RPC APIs and accepts all RPC interfaces and hosts, as well as all IPFS hosts. 
-- `insecure`: uses the Mainnet default configuration, plus sets the flag `no_consensus`, allows all RPC APIs and accepts all RPC interfaces and hosts, as well as all IPFS hosts. 
-- `mining`: uses the Mainnet default configuration, plus increases the number of peers to min 50 and max 100, it disables the Dapps and IPC interface. It forces the sealing of blocks with a minimum of 4 seconds interval, forces the reseal for any new transaction (external or local), reduces the transaction queue size to 2048 while increasing the cache size to 256 MB and setting the `trace` logging level for the `miner` and `own_tx` modules.
-- `non-standard-ports`: sets the client to listen to the port 30305 and 8645 for RPC connections.
-
-## CLI Options for Parity Ethereum client
-```bash
-Operating Options:
```
Comment (5chdn) [inline Configuring-Parity-Ethereum.md]: I like the old format more (sorry) :laughing:

**Comment by joshua-mir** [comment]
I'll fix things manually here before fixing the js stuff, since this is generated per-release.

File: Configuring-Parity-Ethereum.md [hunk 7]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (Tbaut) [inline Configuring-Parity-Ethereum.md]: That prob. doesn't break anything, but it's not needed either :)

File: Configuring-Parity-Ethereum.md [hunk 8]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
-mode = "last"
+```toml[parity]
+chain = foundation
+mode = last
 mode_timeout = 300
 mode_alarm = 3600
-auto_update = "critical"
-release_track = "current"
-public_node = false
+auto_update = critical
```
Comment (Tbaut) [inline Configuring-Parity-Ethereum.md]: Quote missing here as well I guess

File: Configuring-Parity-Ethereum.md [hunk 9]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
 no_consensus = false
-no_persistent_txqueue = false
-
-chain = "homestead"
-base_path = "$HOME/.local/share/io.parity.ethereum"
-db_path = "$HOME/.local/share/io.parity.ethereum/chains"
-keys_path = "$HOME/.local/share/io.parity.ethereum/keys"
+base_path = $BASE
+db_path = $LOCAL/chains
+keys_path = $BASE/keys
```
Comment (Tbaut) [inline Configuring-Parity-Ethereum.md]: quote missing

File: Configuring-Parity-Ethereum.md [hunk 10]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
-password = ["/home/user/.safe/password.file"]
+unlock = [""]
+password = [""]
 keys_iterations = 10240
+refresh_time = 5
+disable_hardware = false
+fast_unlock = false
+
+[ui]
+path = $BASE/signer
```
Comment (Tbaut) [inline Configuring-Parity-Ethereum.md]: quote missing

File: Configuring-Parity-Ethereum.md [hunk 11]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
+allow_ips = all
+no_serve_light = false
+interface = all
+condition = undefined
 
 [rpc]
 disable = false
 port = 8545
-interface = "local"
+interface = local
```
Comment (Tbaut) [inline Configuring-Parity-Ethereum.md]: quote missing?

File: Configuring-Parity-Ethereum.md [hunk 12]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
+max_payload = null
+experimental_rpcs = false
 
 [websockets]
 disable = false
 port = 8546
-interface = "local"
-origins = ["none"]
-apis = ["web3", "eth", "pubsub", "net", "parity", "parity_pubsub", "traces", "rpc", "shh", "shh_pubsub"]
+interface = local
```
Comment (Tbaut) [inline Configuring-Parity-Ethereum.md]: quote missing?

File: Configuring-Parity-Ethereum.md [hunk 13]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
+nodes = [""]
 port = 8083
-path = "$HOME/.local/share/io.parity.ethereum/secretstore"
+interface = local
+http_port = 8082
+http_interface = local
+path = $BASE/secretstore
+disable_http = false
+disable_auto_migrate = false
+acl_contract = registry
```
Comment (Tbaut) [inline Configuring-Parity-Ethereum.md]: quote missing as well as for the "interfaces" and "path" above.

File: Configuring-Parity-Ethereum.md [hunk 14]
```diff
@@ -15,905 +16,2370 @@ mode_timeout = 500
...
+service_contract = null
+service_contract_srv_gen = null
+service_contract_srv_retr = null
+service_contract_doc_store = null
+service_contract_doc_sretr = null
+self_secret = null
+admin_public = null
+server_set_contract = registry
```
Comment (Tbaut) [inline Configuring-Parity-Ethereum.md]: quote missing?

Comment (5chdn) [reply Configuring-Parity-Ethereum.md]: :rofl:

Comment (joshua-mir) [reply Configuring-Parity-Ethereum.md]: I no longer really see the point in manually fixing and merging this - should probably fix all of this stuff properly at the source

**Comment by rakita** [comment]
This seems obsolete, I will just clean it up.

**Pull request closed without merging**